lib/irb/cmd/chws.rb

```diff
@@ -26,6 +25,7 @@ def execute(*obj)
 
     class ChangeWorkspace < Nop
       def execute(*obj)
+        require_relative "../ext/change-ws"
```

Comment:
Unlike `"../ext/multi-irb"`, I'm not sure if we need this. This doesn't override anything, and it impacts only IRB implementation, not the actual context users would eval expressions on IIUC.